### ISSUE TITLE
Parse error when using "in"

### ISSUE THREAD

**Issue by oblitum** [taozhi8833998/node-sql-parser#283]
**Describe the bug**

For the following SQL:

```sql
SELECT position(', ' in 'a, b');
```

I get this error:

```
Expected "#", "$", "(", "--", "/*", "@", "@@", or [ \t\n\r] but "'" found.
```

**Database Engine**

PostgreSQL.

**To Reproduce**
-the SQL that be parsed: `SELECT position(', ' in 'a, b');`
-the node-sql-parser version: `1.9.0`
-the node version: `14.0.0`

**Expected behavior**

No error.

**Comment by taozhi8833998** [comment]
I have released `1.9.1` version to fix the issue, try it.

**Comment by oblitum** [comment]
It's fixed, thanks!
